test/e2e_node/criproxy_test.go

```diff
@@ -91,23 +89,16 @@ var _ = SIGDescribe(feature.CriProxy, framework.WithSerial(), func() {
 			}
 		})
 
-		ginkgo.AfterEach(func() {
-			err := resetCRIProxyInjector(e2eCriProxy)
-			framework.ExpectNoError(err)
-		})
-
```

Comment:
In the other occurrences you are replacing this block by a ginkgo.DeferCleanup but not here, any special reason? or is just an oversight?

Reply:
Oversight, done